persistence/persistence-mybatis-test/src/test/resources/testSql.sql

```diff
@@ -17,6 +17,9 @@ DROP TABLE IF EXISTS "genetic_profile";
 DROP TABLE IF EXISTS "gene";
 DROP TABLE IF EXISTS "cancer_study";
 DROP TABLE IF EXISTS "type_of_cancer";
+DROP TABLE IF EXISTS "gene_panel";
+DROP TABLE IF EXISTS "gene_panel_list";
```

Comment:
The DROP TABLE line for gene_panel_list is still below the DROP TABLE line for gene
The DROP TABLE line for gene_panel_profile_map is still below the DROP TABLE line for sample.
Move this line upwards by about 6 lines (above sample) :
    DROP TABLE IF EXISTS "gene_panel_profile_map";
Move this line upwards by about 4 lines: (above gene)
    DROP TABLE IF EXISTS "gene_panel_list"